book/website/reproducible-research/testing/testing-checklist.md

```diff
@@ -1,8 +1,9 @@
 # Checklist for Code Testing
 
-This checklist contains a lot of items. As [mentioned](#Write_tests_any_tests) it is far better to do some of the items than none of them. Do not be discouraged if this list of tasks seems insurmountable.
+This checklist contains a lot of items. As [mentioned](#Write_tests_any_tests) before, it is far better to do some of the items than none of them. Do not be discouraged if this list of tasks seems insurmountable.
 
-## Writing tests
+<a name="Writing_tests"></a>
+### Writing tests
```

Comment:
Why the extra title level? There does not seem to be a ## title in this document?

Reply:
Fixed -- I think it had ### in the previous PR, good catch :-)